### ISSUE TITLE
NVDA is not announcing the search results.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14515]
### Steps to reproduce:
1. Click on the link https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fcodesandbox.io%2fs%2fdemo-app-7i0c9y&data=05%7c01%7cv-roopesht%40microsoft.com%7ce0374aa6a05d41a8b79708dae8adfd0d%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638078131221355032%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=kUaaknIh%2B%2f4nFJe%2B6jpagbuPvTO6v0oZgK2JSjFA0DI%3D&reserved=0
2. Run NVDA in (Scan off mode)
3. Tab to Browser section.
4. Tab to Search box and enter letter “R” to search and check if screen reader is announcing the number of results found.

### Actual behavior:
As we search the results in the search filed NVDA not announcing the number of results found.

### Expected behavior:
Once the search happens the NVDA should be announcing the number of results found with that search.

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
N/A

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- Please fill out the issue template fully.
- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
- Please update the steps to reflect NVDA and the sample. NVDA does not have a "scan mode".

Kind Regards,
NV Access Software Developers
